### ISSUE TITLE
Rename fonts inside font manager

### ISSUE THREAD

**Issue by reesericci** [FontManager/font-manager#275]
**Is your feature request related to a problem? Please describe.**
I would like to be able to rename fonts to avoid conflicts and/or fixing font names that are rediculous.

**Describe the solution you'd like**
I want to be able to right click, hit rename and rename the font. It should also be editable inside the properties pane.

**Describe alternatives you've considered**
I can't think of any alternatives.

**Additional context**
One of the fonts I installed had a font name of .   I'm not even kidding, the font is completely broken inside figma because of this.
![image](https://user-images.githubusercontent.com/19589006/152544406-9bb90254-3332-45e3-b6b6-71f428a2f30e.png)

**Comment by JerryCasiano** [comment]
I agree, I also think it would be nice to be able to edit the metadata directly from the application.

Unfortunately, I also believe there are quite a few potential pitfalls here...

But probably nothing that can't be worked around. 

I considered it before but set it aside because there were still a lot of basics missing, at this point I think we're doing pretty good as far as basic features so I'm setting this to a 1.0 milestone but will probably work on it before then. It will likely still be a while though.

Thanks for your interest.
